<ui version="4.0" >
 <class>eDTGuiClass</class>
 <widget class="QMainWindow" name="eDTGuiClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>eDT</string>
  </property>
  <property name="iconSize" >
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="toolButtonStyle" >
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="dockNestingEnabled" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <widget class="QStackedWidget" name="stackedWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>595</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex" >
     <number>6</number>
    </property>
    <widget class="QWidget" name="pg_presentation" >
     <widget class="QPushButton" name="pb_Analyse_nl" >
      <property name="geometry" >
       <rect>
        <x>430</x>
        <y>340</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Analyseren</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_Analyse_fr" >
      <property name="geometry" >
       <rect>
        <x>300</x>
        <y>340</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Analyser</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_Presentation" >
      <property name="geometry" >
       <rect>
        <x>40</x>
        <y>130</y>
        <width>141</width>
        <height>128</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="pixmap" >
       <pixmap resource="edtgui.qrc" >:/Resources/Resources/ICO_CARD_EID_TOOLS_128x128.png</pixmap>
      </property>
      <property name="openExternalLinks" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_title" >
      <property name="geometry" >
       <rect>
        <x>200</x>
        <y>170</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>eID Middleware Diagnoseprogramma</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_title_2" >
      <property name="geometry" >
       <rect>
        <x>200</x>
        <y>200</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>eID Middleware Outil de Diagnostique</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_version" >
      <property name="geometry" >
       <rect>
        <x>40</x>
        <y>260</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>eDT : </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pg_information" >
     <widget class="QPushButton" name="pb_Continue_2" >
      <property name="geometry" >
       <rect>
        <x>430</x>
        <y>340</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Start</string>
      </property>
      <property name="icon" >
       <iconset resource="edtgui.qrc" >:/Resources/Resources/wizarrow_right.png</iconset>
      </property>
      <property name="iconSize" >
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_instructions" >
      <property name="geometry" >
       <rect>
        <x>70</x>
        <y>70</y>
        <width>471</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text" >
       <string>instructions</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pg_progress" >
     <widget class="QProgressBar" name="progressBar_Overall" >
      <property name="geometry" >
       <rect>
        <x>180</x>
        <y>210</y>
        <width>351</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value" >
       <number>0</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_Test" >
      <property name="geometry" >
       <rect>
        <x>180</x>
        <y>170</y>
        <width>351</width>
        <height>23</height>
       </rect>
      </property>
      <property name="maximum" >
       <number>0</number>
      </property>
      <property name="value" >
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_Cancel" >
      <property name="geometry" >
       <rect>
        <x>300</x>
        <y>340</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_Continue" >
      <property name="geometry" >
       <rect>
        <x>430</x>
        <y>340</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Continue</string>
      </property>
      <property name="icon" >
       <iconset resource="edtgui.qrc" >:/Resources/Resources/wizarrow_right.png</iconset>
      </property>
      <property name="iconSize" >
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>110</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="pixmap" >
       <pixmap resource="edtgui.qrc" >:/Resources/Resources/ICO_CARD_EID_DIAGNOSIS_128x128.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_testName" >
      <property name="geometry" >
       <rect>
        <x>180</x>
        <y>110</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_testNameHead" >
      <property name="geometry" >
       <rect>
        <x>180</x>
        <y>90</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>Running: </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pg_signing" >
     <widget class="QPushButton" name="pb_testAuthentication" >
      <property name="geometry" >
       <rect>
        <x>300</x>
        <y>340</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Authenticate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_Continue_3" >
      <property name="geometry" >
       <rect>
        <x>430</x>
        <y>340</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Continue</string>
      </property>
      <property name="icon" >
       <iconset resource="edtgui.qrc" >:/Resources/Resources/wizarrow_right.png</iconset>
      </property>
      <property name="iconSize" >
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_2" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>120</y>
        <width>128</width>
        <height>128</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="pixmap" >
       <pixmap resource="edtgui.qrc" >:/Resources/Resources/ICO_CARD_EID_DIGSIG_128x128.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_signing" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>70</y>
        <width>361</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text" >
       <string>Signing instructions</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_signing" >
      <property name="geometry" >
       <rect>
        <x>180</x>
        <y>210</y>
        <width>351</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value" >
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_testSignature" >
      <property name="geometry" >
       <rect>
        <x>180</x>
        <y>180</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text" >
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pg_Summary" >
     <widget class="QLabel" name="lbl_Status" >
      <property name="geometry" >
       <rect>
        <x>220</x>
        <y>20</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text" >
       <string>Succeeded</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_SaveReport" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>340</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_Details" >
      <property name="geometry" >
       <rect>
        <x>40</x>
        <y>340</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Details</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_Diag" >
      <property name="geometry" >
       <rect>
        <x>430</x>
        <y>340</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Diagnostics</string>
      </property>
      <property name="icon" >
       <iconset resource="edtgui.qrc" >:/Resources/Resources/wizarrow_right.png</iconset>
      </property>
      <property name="iconSize" >
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
     </widget>
     <widget class="QTableWidget" name="tbl_Summary" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>70</y>
        <width>581</width>
        <height>261</height>
       </rect>
      </property>
      <property name="editTriggers" >
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="selectionMode" >
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid" >
       <bool>false</bool>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled" >
       <bool>false</bool>
      </property>
      <property name="rowCount" >
       <number>11</number>
      </property>
      <property name="columnCount" >
       <number>2</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
     </widget>
    </widget>
    <widget class="QWidget" name="pg_Details" >
     <widget class="QTextEdit" name="te_Report" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>70</y>
        <width>585</width>
        <height>261</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <family>Courier New</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="readOnly" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_Status_2" >
      <property name="geometry" >
       <rect>
        <x>220</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text" >
       <string>Succeeded</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_SaveReport_2" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>340</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_Summary" >
      <property name="geometry" >
       <rect>
        <x>40</x>
        <y>340</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Summary</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_Diag2" >
      <property name="geometry" >
       <rect>
        <x>430</x>
        <y>340</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Diagnostics</string>
      </property>
      <property name="icon" >
       <iconset resource="edtgui.qrc" >:/Resources/Resources/wizarrow_right.png</iconset>
      </property>
      <property name="iconSize" >
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pg_diagnostics" >
     <widget class="QLabel" name="label_did_it_solve" >
      <property name="geometry" >
       <rect>
        <x>30</x>
        <y>318</y>
        <width>531</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Did the suggestion solve your problem?</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_needhelp" >
      <property name="geometry" >
       <rect>
        <x>540</x>
        <y>340</y>
        <width>23</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text" >
       <string>?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_solved" >
      <property name="geometry" >
       <rect>
        <x>320</x>
        <y>340</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Yes, it did!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_not_solved" >
      <property name="geometry" >
       <rect>
        <x>430</x>
        <y>340</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>No,it didn't.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_Summary_2" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>340</y>
        <width>27</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="icon" >
       <iconset resource="edtgui.qrc" >:/Resources/Resources/wizarrow_left.png</iconset>
      </property>
      <property name="iconSize" >
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_Quit" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>340</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Quit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_SaveSend" >
      <property name="geometry" >
       <rect>
        <x>50</x>
        <y>340</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text" >
       <string>Save</string>
      </property>
     </widget>
     <widget class="QToolBox" name="toolBox" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>30</y>
        <width>551</width>
        <height>271</height>
       </rect>
      </property>
      <property name="currentIndex" >
       <number>1</number>
      </property>
      <widget class="QWidget" name="page" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>551</width>
         <height>215</height>
        </rect>
       </property>
       <attribute name="label" >
        <string>Diagnostic:</string>
       </attribute>
       <widget class="QFrame" name="frame_3" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>551</width>
          <height>221</height>
         </rect>
        </property>
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <widget class="QLabel" name="lbl_details" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>10</y>
           <width>531</width>
           <height>201</height>
          </rect>
         </property>
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="Text" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Text" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Text" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>118</red>
               <green>116</green>
               <blue>108</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font" >
          <font>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="text" >
          <string>details</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="textInteractionFlags" >
          <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::NoTextInteraction|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="suggestion" >
       <attribute name="label" >
        <string>Suggestion:</string>
       </attribute>
       <widget class="QFrame" name="frame" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>551</width>
          <height>211</height>
         </rect>
        </property>
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth" >
         <number>1</number>
        </property>
        <property name="midLineWidth" >
         <number>0</number>
        </property>
        <widget class="QLabel" name="lbl_remedy" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>10</y>
           <width>531</width>
           <height>191</height>
          </rect>
         </property>
         <property name="font" >
          <font>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>suggestion</string>
         </property>
         <property name="textFormat" >
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags" >
          <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::NoTextInteraction|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="lbl_which_diag" >
      <property name="geometry" >
       <rect>
        <x>510</x>
        <y>0</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="pb_next" >
      <property name="geometry" >
       <rect>
        <x>544</x>
        <y>2</y>
        <width>19</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="arrowType" >
       <enum>Qt::RightArrow</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="pb_previous" >
      <property name="geometry" >
       <rect>
        <x>497</x>
        <y>2</y>
        <width>19</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="arrowType" >
       <enum>Qt::LeftArrow</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar" />
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="edtgui.qrc" />
 </resources>
 <connections/>
</ui>
